Stetskyi_Homework_6/DataStructures/DataStructures/Tasks/DoublyLinkedList.cs

```diff
@@ -5,43 +5,118 @@
 
 namespace Tasks
 {
-    public class DoublyLinkedList<T> : IDoublyLinkedList<T>
+    public class DoublyLinkedList<T> : IDoublyLinkedList<T>, IEnumerator<T>, IEnumerator
```

Comment:
@IhorStetskyi, there is no need IEnumerator. Now IEnumerator<T> already inherited IEnumerator.

Reply:
IEnumerator removed